estate/models/estate_property.py

```diff
@@ -0,0 +1,32 @@
+from odoo import models, fields
+
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "Real Estate Property"
+
+    name = fields.Char(required=True, string="Title")
+    description = fields.Text(string="Description")
+    postcode = fields.Char(string="Postcode")
+    date_availability = fields.Date(default=fields.Date.add(fields.Date.today(), days=90), copy=False, string="Available From")
```

Comment:
Not all months consist of  30 days, so to be more precise you can use `months` option.
```suggestion
    date_availability = fields.Date(default=fields.Date.add(fields.Date.today(), months=3), copy=False, string="Available From")
```